marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-clear-user-data.json

```diff
@@ -2,13 +2,20 @@
   "role-name": "hub-central-clear-user-data",
   "description": "This temporarily inherits data-hub-developer so it has the necessary privileges to use Data Hub and to insert artifacts. Will rework this once the new 5.3.0 roles are available.",
   "role": [
+    "hub-central-user",
     "data-hub-developer",
     "data-hub-entity-model-reader",
     "data-hub-flow-reader",
     "data-hub-load-data-reader",
     "data-hub-mapping-reader",
     "data-hub-match-merge-reader",
-    "data-hub-step-definition-reader"
+    "data-hub-step-definition-reader",
+    "data-hub-entity-model-writer",
```

Comment:
I don't think the writer roles are actually needed, as they don't confer any privileges. And technically, we don't want this role to be able to overwrite any of these things - the role only needs to be able to clear the database and then do an initial insert of them.